inst/extdata/gcam-usa/NREL_us_re_capacity_factors.csv

```diff
@@ -0,0 +1,56 @@
+# File: NREL_us_re_capacity_factors.csv,,,,,,
+# Title: Capacity factors by state by renewable technology,,,,,,
+# Units: Unitless,,,,,,
+State,Urban_Utility_scale_PV,Rural_Utility_scale_PV,Rooftop_PV,CSP,Onshore_Wind,Offshore_Wind
```

Comment:
OK, I'll assume this won't hold up the merge unless you tell me otherwise, then.